infercnvpy/tl/_copykat.py

```diff
@@ -0,0 +1,106 @@
+import infercnvpy as cnv
+import numpy as np
+import pandas as pd
+import scipy
+from scipy.sparse import issparse
+from anndata import AnnData
+from scanpy import logging, AnnData
+
+def copyKAT(
```

Comment:
`copykat` neets to be exported to `infercnvpy/tl/__init__.py` to be accessible through the API.

Reply:
done